src/Toast.js

```diff
@@ -33,11 +30,11 @@ function Toast(props) {
   const {
     className,
     cssModule,
-    tag: Tag,
-    isOpen,
+    tag: Tag = 'div',
+    isOpen = true,
     children,
     transition,
```

Comment:
We can add `transition` also as a default parameter right? Is there any issue in doing so and removing `Toast.defaultProps`?

Reply:
I was lazy and avoided this as it is not a simple case. I think it needs a deep merge because there are nested objects. Simply doing 
```
transition = {
    ...Fade.defaultProps,
    unmountOnExit: true,
  }
```

would not allow for, e.g. prop of `{transition: {appear: false}}`. `transition` wouldn't be empty, so it would lose all the other defaultProp values

Reply:
> I was lazy and avoided this as it is not a simple case. I think it needs a deep merge because there are nested objects. Simply doing
> 
> ```
> transition = {
>     ...Fade.defaultProps,
>     unmountOnExit: true,
>   }
> ```
> 
> would not allow for, e.g. prop of `{transition: {appear: false}}`. `transition` wouldn't be empty, so it would lose all the other defaultProp values

That is how default props work now; defaultProps doesn't merge objects, it just uses the default if the prop is `undefined`. Later in the code, a shallow merge is done to add any missing default values on the object root (which, looking at the Fade component, is all that really matters.) Maybe `unmountOnExit: true,` should be added there (but that would technically be a change to how it currently would work.)

Reply:
@TheSharpieOne ah I did not realise that (I've never used `defaultProps` myself). I've updated in this instance. Thank you